.babelrc

```diff
@@ -1,26 +1,59 @@
 {
-  "presets": [["env", {
-    "targets": {
-      "node": 4,
-      "browsers": [
-        "ie 9",
-        "ios 9",
-        "last 2 chrome versions",
-        "last 2 edge versions",
-        "last 2 firefox versions",
+  "env": {
```

Comment:
_Note_: The babel config merging strategy does not allow the common plugins to be defined at the top level. I'm investigating this babel bug and will file an issue on that project when I have a good repro.